data_dir/c-s_uda_udf.yaml

```diff
@@ -0,0 +1,74 @@
+# Keyspace Name
+keyspace: ks
+
+# The CQL for creating a keyspace (optional if it already exists)
+keyspace_definition: |
+  CREATE KEYSPACE ks WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
+# Table name
+table: uda_udf
+
+# The CQL for creating a table you wish to stress (optional if it already exists)
+table_definition: |
+  CREATE TABLE uda_udf (
+        pk timeuuid,
+        ck_int int,
+        c1_timeuuid timeuuid,
+        c2_int int,
+        c3_int int,
+        c4_bigint bigint,
+        c5_double double,
+        c6_float float,
+        c7_text text,
+        PRIMARY KEY(pk, ck_int)
+  ) WITH compaction = { 'class':'SizeTieredCompactionStrategy' }
```

Comment:
why specifying 'SizeTieredCompactionStrategy' and not leaving the default, so for enterprise it'll use ICS?

Reply:
I'm actually not sure why it was set that way. Either way, I got rid of it -we can go with defaults here.